fdbclient/ManagementAPI.actor.cpp

```diff
@@ -1116,9 +1116,10 @@ struct AutoQuorumChange final : IQuorumChange {
 		self->addDesiredWorkers(chosen, workers, desiredCount, excluded);
 
 		if (chosen.size() < desiredCount) {
-			if (chosen.size() < oldCoordinators.size()) {
+			if (chosen.size() < oldCoordinators.size() || chosen.size() == 0) {
```

Comment:
Move this before the "<" condition, just in case oldCoordinators.size() is also 0.

Nit: chosen.empty() is easier to read.

Reply:
Right right.